scripts/run.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import os
+import sys
+
+# pylint: disable=wrong-import-position
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from allennlp.run import train_model_from_file
+from allennlp.common.checks import ensure_pythonhashseed_set
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def main():
+    usage = 'USAGE: train.py [param_file] [train]'
```

Comment:
I don't understand what the `[train]` parameter is here? from the code it looks like it's supposed to be `"train"`?  

anyway, I have a grand plan to unify all of the command-line tools, so it will probably get revamped then anyway